base-notebook/start-notebook.sh

```diff
@@ -15,5 +15,6 @@ if [[ ! -z "${JUPYTERHUB_API_TOKEN}" ]]; then
 elif [[ ! -z "${JUPYTER_ENABLE_LAB}" ]]; then
     . /usr/local/bin/start.sh $wrapper jupyter lab "$@"
 else
+    echo "WARN: Jupyter Notebook deprecation notice https://github.com/jupyter/notebook#notice."
```

Comment:
Should this point to the stacks readme section you added instead, where we have more detail about the flag and the link upstream?

Reply:
Yes it's much better I will change It.

Reply:
Changes done.